op-batcher/cmd/main.go

```diff
@@ -4,13 +4,14 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/urfave/cli"
-
+	op_aws_sdk "github.com/ethereum-optimism/optimism/op-aws-sdk"
 	"github.com/ethereum-optimism/optimism/op-batcher/batcher"
 	"github.com/ethereum-optimism/optimism/op-batcher/cmd/doc"
 	"github.com/ethereum-optimism/optimism/op-batcher/flags"
 	oplog "github.com/ethereum-optimism/optimism/op-service/log"
 	"github.com/ethereum/go-ethereum/log"
+	"github.com/urfave/cli"
+	"golang.org/x/net/context"
```

Comment:
`golang.org/x/net/context` can be replaced by `context` after version golang 1.7. I recommend keeping consistent with the style of other codes and using `context` instead of `golang.org/x/net/context`.